pwndbg/commands/heap.py

```diff
@@ -95,6 +95,53 @@ def format_bin(bins, verbose=False, offset=None):
     return result
 
 
+parser = argparse.ArgumentParser()
+parser.description = "Template heap command. You can ignore this."
+parser.add_argument("addr", type=int, help="Address of a chunk header.")
+
+
+@pwndbg.commands.ArgparsedCommand(parser)
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithResolvedHeapSyms
+@pwndbg.commands.OnlyWhenHeapIsInitialized
+def template_heap_command(addr):
+    """Template heap command with example uses of pwndbg's heap classes."""
+
+    # `addr` is a gdb.Value of one of several types, depending on how the user invoked this command
+    # e.g. template_heap_command <address> vs. template_heap_command <symbol>
+
+    # The `Chunk` class abstracts away many heap & gdb module internals.
+    chunk = pwndbg.heap.ptmalloc.Chunk(addr)
+
+    print(f"chunk.address: 0x{chunk.address:02x}")
```

Comment:
I think this PR is great, the only tiny potential issue I can find is: should we use `f-strings` syntax?
I like `f-strings` and it's convenient, but for the compatibility, maybe we should not use it until we fully deprecate Python<3.6 support I guess?
(I saw the comment from @disconnect3d about the Python version we want to support here: https://github.com/pwndbg/pwndbg/pull/1063#issuecomment-1219942748)